lib/fail.js

```diff
@@ -3,6 +3,7 @@ var match = require('pattern-match');
 function ValidationError(message, stack) {
     Error.call(this, message);
     this.stack = stack;
+    this.message = message;
```

Comment:
I think so. Try this:

``` js
function MyError(message) {
  Error.call(this, message);
}

MyError.prototype = Object.create(Error.prototype);
MyError.prototype.constructor = MyError;

try {
  throw new MyError('hello');  
} catch (e) {
  console.log('Error message: ' + e.message);  
}
```

[JSBin demo](http://jsbin.com/gijeyisifa/1/edit?js,console)

The message property is not set, even after you call `Error.call()` (tried in FireFox and Chrome).